Error: HTTPConnectionPool(host='service2', port=8000): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f435e7659d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2 2023-09-22T09:26:00.348245z service2:8000
3 2023-09-22T09:26:02.522189z service2:8000
4 2023-09-22T09:26:04.530155z service2:8000
5 2023-09-22T09:26:06.542049z service2:8000
6 2023-09-22T09:26:08.557000z service2:8000
7 2023-09-22T09:26:10.573384z service2:8000
8 2023-09-22T09:26:12.591852z service2:8000
9 2023-09-22T09:26:14.608204z service2:8000
10 2023-09-22T09:26:16.628152z service2:8000
11 2023-09-22T09:26:18.645217z service2:8000
12 2023-09-22T09:26:20.660432z service2:8000
13 2023-09-22T09:26:22.674949z service2:8000
14 2023-09-22T09:26:24.692929z service2:8000
15 2023-09-22T09:26:26.709882z service2:8000
16 2023-09-22T09:26:28.727918z service2:8000
17 2023-09-22T09:26:30.743448z service2:8000
18 2023-09-22T09:26:32.758039z service2:8000
19 2023-09-22T09:26:34.779568z service2:8000
20 2023-09-22T09:26:36.796767z service2:8000
